pagopa/io-app
# Simplify definitions generation

## Short description
Use the "*" feature to generate all the definitions.

## List of changes proposed in this pull request
Use the "*" feature to generate the definitions instead of manually include each generate script.

## How to test
Try to use `yarn generate` instead of `yarn generate:all` and check all the definitions are still generated.
